README.rst

```diff
@@ -32,16 +27,15 @@ is installed by `pip`.
 Usage
 -----
 
+.. code-block:: text
```

Comment:
Mh, according to [the docs](https://build-me-the-docs-please.readthedocs.io/en/latest/Using_Sphinx/ShowingCodeExamplesInSphinx.html#pygments-lexers) this should probably be `.. code-block:: bash`